it/repositories/sifting/SiftingRepositorySpec.scala

```diff
@@ -0,0 +1,88 @@
+package repositories.sifting
+
+import model.EvaluationResults.Green
+import model.Phase3TestProfileExamples.phase3TestWithResult
+import model.ProgressStatuses.PHASE3_TESTS_PASSED
+import model.persisted.{ PassmarkEvaluation, SchemeEvaluationResult }
+import model.{ ApplicationStatus, SchemeId }
+import repositories.onlinetesting.Phase2EvaluationMongoRepositorySpec.phase2TestWithResult
+import repositories.{ CollectionNames, CommonRepository }
+import testkit.MongoRepositorySpec
+
+class SiftingRepositorySpec extends MongoRepositorySpec with CommonRepository {
+
+  override val collectionName = CollectionNames.APPLICATION
+
+  def repository = new SiftingMongoRepository()
+
+  private val userId = "123"
+
+  private val Commercial: SchemeId = SchemeId("Commercial")
+  private val European: SchemeId = SchemeId("European")
+  private val Sdip: SchemeId = SchemeId("Sdip")
+
+  "Sifting repository" should {
```

Comment:
There are different styles in the source code but I prefer the structure it in this way, forget about: "Sifting repository" should {
because we are in SiftingRepository already and use something like:
"find eligible candidates" should {
   "return candidates in a particular state when whatever condition" in { ...